core/server/common/src/main/java/alluxio/master/journal/raft/SnapshotReplicationManager.java

```diff
@@ -519,7 +524,7 @@ private boolean requestData() {
   }
 
   @VisibleForTesting
-  synchronized RaftJournalServiceClient getJournalServiceClient()
+  synchronized RaftJournalServiceClient createJournalServiceClient()
```

Comment:
This method was renamed after Rongrong's presentation on Java anti-patterns, notably functions called `get` that are not getters.